cartographer_ros/cartographer_ros/offline_node.cc

```diff
@@ -122,8 +122,9 @@ void RunOfflineNode(const MapBuilderFactory& map_builder_factory) {
   tf2_ros::Buffer tf_buffer;
 
   std::vector<geometry_msgs::TransformStamped> urdf_transforms;
-  for (const std::string& urdf_filename :
-       cartographer_ros::SplitString(FLAGS_urdf_filenames, ',')) {
+  const std::vector<std::string> urdf_filenames = 
+      absl::StrSplit(FLAGS_urdf_filenames, ',');
+  for (const auto& urdf_filename : urdf_filenames) {
```

Comment:
We have to be explicit here, otherwise we'll get `absl::string_view` in the loop.